wave_vision/config/descriptor/brisk.yaml

```diff
@@ -0,0 +1,25 @@
+# Configuration parameters for BRISK Descriptor Extractor
+
+# radiusList defines the radius of each subsequent circle (in pixels). All
+# numbers must be positive. Cannot be empty.
+#
+# Default: radiusList: [0.0, 2.465, 4.165, 6.29, 9.18]
+radiusList: [0.0, 2.465, 4.165, 6.29, 9.18]
+
+# numberList defines the number of points in each circle. Must be the same
+# size as radiusList. All numbers must be positive. Cannot be empty.
+#
+# Default: numberList: [1, 10, 14, 15, 20]
```

Comment:
I've set it up so that if the user doesn't include any parameters, an exception is thrown. So hopefully that will realize they should use something!